Client/Components/PageNotifications.razor.cs

```diff
@@ -1,17 +1,24 @@
 ﻿namespace BlazorRepl.Client.Components
 {
     using System.Collections.Generic;
+    using System.Timers;
     using BlazorRepl.Client.Models;
 
     public partial class PageNotifications
```

Comment:
should we dispose timers when the component is disposed?

Reply:
Since it's a global component (defined as cascading in `MainLayout`) there won't be any case where it would be disposed and the app be still running to execute the disposal.

Reply:
What if we go to `/getting-started` or refresh. IMO we should release all the memory

Reply:
All pages that we have except `UserPage` are using the `MainLayout` so won't be an issue. Will add the disposal though so we are always safe.